docs/user-interface/skills_knowledge_contributions.md

```diff
@@ -0,0 +1,83 @@
+
+For tips on writing Skills & Knowledge contributions, please visit the documentation under the [Taxonomy](/taxonomy/) heading.
+
+## Skill Contributions
+
+### Author Information
+
+Enter your github email address, and your full name, which will be tied to your contribution.
+
+### Skill Information
+
+Give a brief description on what skill you would like to teach.
+
+### Taxonomy Directory Path
+
+Specify where in the taxonomy this skill would belong in. As the taxonomy fills, it will become more complex, make sure to give a concise location to keep the taxonomy organised.
+
+### Seed Examples
+
+For a skill contribution, the seed example only requires a question and answer, but depending on what skill you wish to add, adding context for a skill may be necessary, For tips on writing Skills contributions, please visit the documentation under the [Skills Guide](/taxonomy/skills/skills_guide) heading.
+
+### Attribution
+
+Linking the source you used for information. Wikipedia articles change overtime, make sure to add the oid from the wikipedia article, which you can find by clicking on `View history`, and select the relevant version.
+
+## Knowledge Contributions
+
+### Author Information
+
+Enter your github email address, and your full name, which will be tied to your contribution.
+
+### Knowledge Information
+
+Give a brief description of what knowledge you are submitting, where you received the information from, and then a detailed outline of the document, mentioning the topics and headings in your topic.
+
+### Taxonomy Directory Path
+
+fy where in the taxonomy this skill would belong in. As the taxonomy fills, it will become more complex, make sure to give a concise location to keep the taxonomy organised.
+
+### Seed Examples
+
+For a Knowledge contribution, A user must provide context, which would usually be a snippet or a heading extrated from the main document. The user must then write at least 3 questions based on the context given, and answer said questions. For tips on writing Knowledge contributions, please visit the documentation under the [Knowledge Guide](/taxonomy/knowledge/guide) heading.
+
+### Document Information
+
+You must prepare a markdown file version of the document you wish to use for the knowledge submission. By dragging and dropping the markdown file into the box, and clicking the submit files button, a forked version of the taxonomy repository will be automatically created on your github profile. 
+
+![UI Knowledge Document Information](../images/user-interface/ui_knowledge_document_info.png)
+
+![Forked Repository Showcase](../images//user-interface/ui_knowledge_repo_created.png)
+
+If you've already uploaded the markdown file to your github, you can switch to manually adding the document, and entering the `commit sha`.
+
```

Comment:
I think it would be helpful if we add the snapshot of the Manually adding the doc details page.